docs/src/introduction.md

```diff
@@ -4,15 +4,28 @@ Polygon Miden's AirScript is designed to make it simple to describe AIR constrai
 
 ## Current Version
 
-Currently, AirScript is on version 0.1, which supports a simple syntax for describing air constraints and generation of Rust code targeting the [Winterfell prover](https://github.com/novifinancial/winterfell).
+Currently, AirScript is on version 0.2, which supports a simple syntax for describing air constraints and generation of Rust code targeting the [Winterfell prover](https://github.com/novifinancial/winterfell).
 
-The simplified version of the language is based on [this discussion](https://github.com/maticnetwork/miden/discussions/254), but only includes the following:
+The language is based on [this discussion](https://github.com/maticnetwork/miden/discussions/254), and includes features to write ~90% of the constraints required for Miden albeit with some limitations.
 
-- declaring trace columns for main and auxiliary traces as a vector (e.g. `main: [a, b, c]` but not `main[a, b, c[2]]`)
-- declaring public inputs where each public input is a named vector (e.g. `stack_inputs: [16]`)
-- declaring periodic columns
-- enforcing boundary constraints for main and auxiliary traces using trace columns, public inputs, and inline scalar constants
-- enforcing transition constraints for main and auxiliary traces using trace columns, periodic columns, and inline scalar constants
+AirScript includes the following features:
+
+- <b>Trace Columns</b>: Users can declare trace columns for main and auxiliary traces as individual columns or groups of columns (e.g. `main: [a, b, c[3], d]` where `a`, `b`, and `d` are single columns and `c` refers to a group of 3 columns)
+
+- <b>Public Inputs</b>: Users can declare public inputs where each public input is a named vector (e.g. `stack_inputs: [16]`)
+
+- <b>Periodic Columns</b>: Users can declare periodic columns (e.g. `k0: [1, 0, 0, 0]`)
+
+- <b>Random Values</b>: Users can define random values provided by the verifier (e.g. `alphas: [x, y[14], z]` or `rand: [16]`)
+
+- <b>Boundary Constraints</b>: Users can enfore boundary constraints on main and auxiliary trace columns using trace columns, public inputs, random values, constants and variables.
```

Comment:
Do we allow referencing trace columns in boundary constraints?

Reply:
Ah, sorry, fixed. It was actually there before and I was going to remove it but then missed it somehow.